internal/cmd/kafka/command_cluster.go

```diff
@@ -2,6 +2,7 @@ package kafka
 
 import (
 	"fmt"
+	presource "github.com/confluentinc/cli/internal/pkg/name-conversions"
```

Comment:
Which IDE are you using? Most IDEs can sort these imports automatically for you, FYI.

Reply:
I'm using Goland. For import sorting type, do we use `goimports`?

Reply:
Yep, `goimports` should get you 95% of the way there. For the other 5% we use `gci` in our linter which unfortunately doesn't have an auto-format tool.